app/initstores.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"net/url"
 
+	"github.com/target/goalert/notice"
+
```

Comment:
nit: remove extra whitespace -- don't forget to run goimports to sort after

Reply:
Interesting, this was the result of `goimports -w app/initstores.go`